Analysis/SLDCorrection/src/SLDCorrection.cc

```diff
@@ -801,7 +801,7 @@ void SLDCorrection::init()
 		m_pTTree1->Branch( "DSVDistanceFromPV" , &m_DSVDistanceFromPV );
 		m_pTTree1->Branch( "Lepton3DImpactParameter" , &m_Lepton3DImpactParameter );
 		m_pTTree1->Branch( "OtherParticle3DImpactParameter" , &m_OtherParticle3DImpactParameter );
-		h_SLDStatus = new TH1I( "SLDStatus" , ";" , 7 , 0 , 7 );
+		h_SLDStatus = new TH1I( "SLDStatus" , ";" , 8 , -1 , 7 );
```

Comment:
I think the number of bins should change here as well? Or was it incorrect before? I suppose the main reason for doing this is to have access to things that haven't been classified into any of the pre-assigned categories? However, the previous version had that already if you use the under-/overflow bins.

Reply:
Before, the values were before 0/7, now -1/7 (throughout the code, floats are inserted, i.e. 0.5 -> 1. I didn't change this). According to [here](https://root.cern.ch/doc/master/classTH1.html) the third argument is the bin count, so it should be fine?

Reply:
Ah right, thanks for the clear up. I was once more confusing the order of arguments in the `TH1I` constructor. But as far as I can see there is no actual need for a negative bin with value `-1`? `h_SLDStatus` is only filled with the values `-0.5`, `0.5` and `1.5` at the moment.